web/src/main/webapp/dataViews/tableView.ts

```diff
@@ -959,6 +964,20 @@ export class TableView extends TSViewBase implements IScrollTarget {
         }
     }
 
+    protected runCountSketch(): void{
+        const columnsShown: IColumnDescription[] = [];
+        const cso: ColumnSortOrientation[] = [];
+        this.getSelectedColNames().forEach((v) => {
+            const colDesc = this.schema.find(v);
```

Comment:
some column names may have been renamed, I hope this handles this case correctly.